packages/typespec-azure-vscode/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "typespec-azure-vscode",
```

Comment:
How do you plan to distribute it initially? Right now it will get published to npm. Make sure to add `private: true` if you don't want to.

Reply:
hello @timotheeguerin `typespec-azure-extension`  will be published in public marketplace. I am not sure whether we need to publish to npm also.  Current `typespec-extension` is published to npm (https://www.npmjs.com/package/typespec-vscode).  Do you know why `typespec-extension` is published to npm? and is it required?
if it is not required, I think `typespec-azure-extension` will not publish to npm, just marketplace only.

Reply:
typespec extension was published to npm first as we couldn't originally publish to the marketplace. So it published to npm and install was done via the `tsp code install` command that downloaded the package and run the vscode command line on the vsix.

Reply:
hello @timotheeguerin do you mean to add the package typespec-azure-vscode to chronus config `versionPolicise`? as following
![image](https://github.com/user-attachments/assets/76c0ec2d-00a9-4d1f-8333-6a42f42f1ccb)

thanks